### PR TITLE
Fixes #2166. Set min-height on text-field

### LINKED ISSUE #2166: "Leave a comment" text area should have a limit when minimized
URL: https://staging.webcompat.com/issues/1378
Firefox Nightly Mobile 60.0a1 (2018-02-18), 
Operating System: Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Prerequisites:**
1. Github account available and signed in.

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/issues/1378
2. Scroll down the page to "Comment" section.
3. Minimize the "Leave a comment" text area.
4. Type in the area and observe text.

**Expected Behavior:**
Text is fully visible. "Leave a comment" text area has a limit when minimized (1-2 rows).

**Actual Behavior:**
Text is not visible.

**Note**
1. Also reproducible on Desktop.
2. Affected area:
```html
<textarea id="newComment" class="form-field text-field light js-Comment-text mousetrap" rows="2" placeholder="Leave a comment" style="height: 134px;"></textarea>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

screenshot-1519054991549

minimizedtextarea

### PR DESCRIPTION
**Pull request by Regaddi**
r? @zoepage

### CODE REVIEW

File: webcompat/static/css/src/form.css [hunk 1]
```diff
@@ -89,6 +89,7 @@
 
 .text-field {
   display: block;
+  min-height: 27px;
```
Comment (zoepage) [inline webcompat/static/css/src/form.css]: This is already a good start to make sure all input fields as well as the text area showing text.

**Comment by zoepage** [comment]
The main issue is here https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/static/css/src/issue.css#L46 and can be seen e.g. https://staging.webcompat.com/issues/1388

When you minimize the textarea, the text disappears as the padding is interfering.

If you set `padding: calc(var(--unit-space) * 2);` to `padding: 0 calc(var(--unit-space) * 2);` it should do the trick :)

**Comment by Regaddi** [comment]
Why is there even another padding on the textarea? I'd tend to remove it completely from issue.css and let the default text-field padding apply 🤔

**Comment by zoepage** [comment]
As far as I remember, it was set for the validation checkmarks.

**Comment by Regaddi** [comment]
So it depends wether you want that padding in the box for design purposes, or not.
Another solution, that should do well (without affecting the padding) is using a calculated `min-height: calc(1em + 40px)` for the `.issue-new-comment .text-field`

**Comment by zoepage** [comment]
I think, just removing that padding on top and bottom will be ok. So we don't introduce more of 



How does this sound for you? :)

**Comment by Regaddi** [comment]
There you go @zoepage. No magic.

**Pull request merged**